src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -164,6 +180,31 @@ def _row_to_doc(row, kbase_collection, load_version, features, tool_genome_map,
     return doc
 
 
+def _update_docs_with_upa_info(res_dict, meta_lookup):
+    # Update original docs with UPA informathion through a meta hashmap
+    
+    for genome_id in res_dict:
+        try:
+            meta_filename = meta_lookup[genome_id]
+        except KeyError as e:
+            raise ValueError('Unable to find genome ID') from e
+        
+        upa_dict = {}
```

Comment:
```suggestion
```

Reply:
Ah... what about at line 202 res_dict[genome_id].update({names.FLD_UPA_MAP: upa_dict})?

upa_dict should be an empty {} if running on GTDB data. That's why I declared upa_dict at line 192.

Reply:
Yep, you're right. Ignore this and the related comment.